lxd/endpoints/network_util.go

```diff
@@ -0,0 +1,47 @@
+package endpoints
+
+import (
+	"net"
+	"os"
+	"regexp"
+)
+
+type networkServerErrorLogWriter struct {
+	proxies []net.IP
+}
+
+// Regex for the log we want to ignore.
+var unwantedLogRegex = regexp.MustCompile(`http: TLS handshake error from ([^\[:]+?|\[([^\]]+?)\]):[0-9]+: .+write: connection reset by peer\s$`)
+
+func (d networkServerErrorLogWriter) Write(p []byte) (int, error) {
+	if d.shouldDiscard(p) {
+		return 0, nil
+	}
+
+	return os.Stderr.Write(p)
```

Comment:
We'll need to strip things a bit and use our logger. os.Stderr doesn't hit our logging API and so would bypass any integration with audit systems and the like.

Reply:
Ok no problem, I'll strip everything before the `http: ...`